monkey/tests/unit_tests/monkey_island/cc/resources/test_agent_logs.py

```diff
@@ -0,0 +1,59 @@
+from http import HTTPStatus
+from uuid import UUID
+
+import pytest
+from tests.common import StubDIContainer
+
+from common.types import AgentID
+from monkey_island.cc.repository import IAgentLogRepository
+
+AGENT_ID_1 = UUID("c0dd10b3-e21a-4da9-9d96-a99c19ebd7c5")
+AGENT_ID_2 = UUID("f811ad00-5a68-4437-bd51-7b5cc1768ad5")
+
+AGENT_LOGS_URL_1 = f"/api/agent-logs/{AGENT_ID_1}"
+AGENT_LOGS_URL_2 = f"/api/agent-logs/{AGENT_ID_2}"
+
+
+class StubAgentLogRepository(IAgentLogRepository):
```

Comment:
This is not a stub. Stubs have no logic. See https://martinfowler.com/bliki/TestDouble.html

I would instead call this `InMemoryAgentLogRepository` and put it with the rest of the `InMemory` repositories.